src/Microsoft.OData.Core/ODataResource.cs

```diff
@@ -186,6 +186,11 @@ public IEnumerable<ODataProperty> Properties
             set { this.properties = value; }
         }
 
+        internal bool IsExpandedEntityProcessed(string name)
+        {

```

Comment:
should have code comments and be moved.

is this even needed? why doesn't the calling code just do resource.MetadataBuilder.IsExpandedEntityProcessed(name)?

Reply:
I thought having this convenient method on ODataResouce would save the indirection underneath and therefore would be beneficial for the user. 
But you are right that it is not strictly needed. Removed.

---
In reply to: [229462320](https://github.com/OData/odata.net/pull/1106#discussion_r229462320) [](ancestors = 229462320)